modules/javafx.graphics/src/main/java/javafx/scene/LightBase.java

```diff
@@ -319,7 +398,7 @@ private void markOwnerDirty() {
     private void markChildrenDirty(Node node) {
         if (node instanceof Shape3D) {
             // Dirty using a lightweight DirtyBits.NODE_DRAWMODE bit
-            NodeHelper.markDirty(((Shape3D) node), DirtyBits.NODE_DRAWMODE);
+            NodeHelper.markDirty(node, DirtyBits.NODE_DRAWMODE);
```

Comment:
I realize that this is an unnecessary cast, but since this is a doc-only change and we are in rampdown for jfx18, it would be best to revert this change.